setup.py

```diff
@@ -9,8 +9,7 @@
     author='jarbasAi',
     install_requires=["ovos_utils>=0.0.12a3",
                       "requests",
-                      "memory-tempfile",
-                      "phoneme_guesser"],
+                      "memory-tempfile"],
```

Comment:
Consider a loose version spec to help version resolution

Reply:
i dont think we have min or max versions for any of those packages...

this change is only removing one dependency, so im calling this one out of scope for this PR